build.yaml

```diff
@@ -1554,6 +1571,52 @@ job_type:
       timeout: 90
       directories_to_delete: []
 
+    run_acceptance_on_GCE_CentOS_7_with_LOOPBACK:
+      at: '0 5 * * *'
+      on_nodes_with_labels: 'aws-centos-7-SELinux-T2Medium'
```

Comment:
Interestingly it doesn't (and in fact can't). This just specifies which slaves the run_acceptance_test code will be run. Presently jenkins has no GCE slaves because teaching jenkins how to spin up GCE instances turns out to be non-trivial. So, this will be running code on AWS slaves that will spin up a flocker cluster on GCE  and run tests against them.

It is sort of subtle, but it is also what we do with rackspace (The assumption is we prefer to use AWS slaves for all jobs unless there is a reason not to).